### PR TITLE
Incorporate changes from the demo

### LINKED ISSUE #56: Setup demo

### PR DESCRIPTION
**Pull request by lukemoore495**

### CODE REVIEW

File: frontend/src/app/alternatives-form/measurements-panel/measurements-panel.component.ts [hunk 1]
```diff
@@ -50,6 +52,8 @@ export class MeasurementsPanelComponent implements OnInit, OnDestroy {
     }
 
     ngOnInit(): void {
+        this.isTopLevel = this.measurementNode?.weight === 1;
```
Comment (lukemoore495) [inline frontend/src/app/alternatives-form/measurements-panel/measurements-panel.component.ts]: This is strange and probably wrong. Pass an isTopLevel flag into the panel instead.

File: frontend/src/app/dash/dash.component.html [hunk 2]
```diff
@@ -28,6 +28,9 @@
                 </mat-option>
             </mat-select>
           </mat-form-field>
+          <button mat-raised-button color="accent" class="button">
```
Comment (lukemoore495) [inline frontend/src/app/dash/dash.component.html]: There should also be a delete button

File: frontend/src/app/hierarchical-view/hierarchical-view.component.html [hunk 3]
```diff
@@ -8,12 +8,16 @@
         <mat-icon>add</mat-icon>
         Create Hierarchy
     </button>
+    <button mat-raised-button color="warn" class="header-button">
+        <mat-icon>delete</mat-icon>
+        Delete Hierarchy
+    </button>
 
     <!-- This is a short term hack to save RRRHierarchy into the db -->
-    <button mat-raised-button color="accent" class="header-button" (click)="createHierarchy()">
```
Comment (lukemoore495) [inline frontend/src/app/hierarchical-view/hierarchical-view.component.html:13]: Uncomment this for convenience's sake.

File: frontend/src/app/nav/nav.component.ts [hunk 4]
```diff
@@ -17,7 +17,7 @@ export class NavComponent implements OnInit {
     hasHierarchies = false;
 
     menuItems = [
-        { title: 'Hierarchical View', route: 'hierarchicalView', icon: 'pageview' },
+        { title: 'Hierarchy Viewer', route: 'hierarchicalView', icon: 'pageview' },
```
Comment (lukemoore495) [inline frontend/src/app/nav/nav.component.ts:20]: Go ahead and rename the component too

**Comment by DavBebawy** [comment]
Approved

**Review by davbebawy — APPROVED** [review]
Approved

**Pull request merged**